data/json/effects_on_condition/npc_eocs/appearance_eocs.json

```diff
@@ -232,6 +232,137 @@
+                    "else": {
+                      "if": { "u_has_trait": "hair_long" },
+                      "then": [ { "u_lose_trait": "hair_long" }, { "u_add_trait": "hair_long", "variant": { "context_val": "color_id" } } ],
+                      "else": {
+                        "if": { "u_has_trait": "hair_long_mohawk" },
+                        "then": [
+                          { "u_lose_trait": "hair_long_mohawk" },
+                          { "u_add_trait": "hair_long_mohawk", "variant": { "context_val": "color_id" } }
+                        ],
+                        "else": {
+                          "if": { "u_has_trait": "hair_mullet" },
+                          "then": [ { "u_lose_trait": "hair_mullet" }, { "u_add_trait": "hair_mullet", "variant": { "context_val": "color_id" } } ],
+                          "else": {
+                            "if": { "u_has_trait": "hair_long_over_eye" },
+                            "then": [
+                              { "u_lose_trait": "hair_long_over_eye" },
+                              { "u_add_trait": "hair_long_over_eye", "variant": { "context_val": "color_id" } }
+                            ],
+                            "else": {
+                              "if": { "or": [ { "u_has_trait": "hair_messy" } ] },
+                              "then": [ { "u_lose_trait": "hair_messy" }, { "u_add_trait": "hair_messy", "variant": { "context_val": "color_id" } } ],
+                              "else": {
+                                "if": { "or": [ { "u_has_trait": "hair_very_long" } ] },
+                                "then": [ { "u_lose_trait": "hair_very_long" }, { "u_add_trait": "hair_very_long", "variant": { "context_val": "color_id" } } ],
+                                "else": {
+                                  "if": { "or": [ { "u_has_trait": "hair_extremely_long" } ] },
+                                  "then": [
+                                    { "u_lose_trait": "hair_extremely_long" },
+                                    { "u_add_trait": "hair_extremely_long", "variant": { "context_val": "color_id" } }
+                                  ],
+                                  "else": {
+                                    "if": { "or": [ { "u_has_trait": "hair_body_length" } ] },
+                                    "then": [
+                                      { "u_lose_trait": "hair_body_length" },
+                                      { "u_add_trait": "hair_body_length", "variant": { "context_val": "color_id" } }
+                                    ]
+                                  }
+                                }
+                              }
+                            }
+                          }
+                        }
+                      }
+                    }
+                  }
+                }
+              }
+            }
+          }
+        }
+      }
+    ]
+  },
```

Comment:
holy stack of closing braces, batman.

Reply:
I don’t know how to fix that, but I’ll look into it.

Reply:
You don't have to fix it. I just think it's funny